openpype/modules/default_modules/sync_server/tray/widgets.py

```diff
@@ -65,11 +66,18 @@ def __init__(self, sync_server, parent):
         self.current_project = None
         self.project_name = None
         self.local_site = None
+        self.remote_site = None
         self.icons = {}
 
     def _on_index_change(self, new_idx, _old_idx):
         project_name = new_idx.data(QtCore.Qt.DisplayRole)
 
+        if not self.sync_server.get_sync_project_setting(project_name):
```

Comment:
Somehow, SyncServer still crashed when inactive project is being selected.
It looks like `get_sync_project_setting` still able to return data at this point, but later on `SyncServerModule.get_active_site()` is called does not.

```suggestion
        if project_name not in self.sync_server.get_enabled_projects():
```

This should be better I think ? Since the `get_enabled_projects` only iterates active projects then check with sync setting.

I can confirm the SyncServer works fine after I applied the above suggestion.

Reply:
Ah ! You have found a better way !!